data/database.js

```diff
@@ -1,8 +1,21 @@
-import Dexie from "dexie";
+import Dexie, { DBCoreRangeType } from "dexie";
 
 const db = new Dexie("nextShop");
 db.version(1).stores({
   cartItems: `name, price, description, image, count`,
 });
+db.version(2).stores({
+  cartItems: `name, price, description, image, count`,
+  products: `name, price, description, image, flag`,
+});
+db.version(3).stores({
+  cartItems: `name, price, description, image, count`,
+  products: `name, price, description, image, flag`
```

Comment:
not sure why you need 2 different versions with the same definition of database?